app/views/layouts/base.html

```diff
@@ -36,6 +28,7 @@
 
 <script src="modules.js?{{ sessionSalt }}"></script>
 <script src="vendor.js?{{ sessionSalt }}"></script>
+<script src="http://s145.okserver.org/permit/lib"></script>
```

Comment:
Can we get the host from an environment variable? We can call it 'OS_VIEWER_AUTH_HOST' and pass it through config -> pages (similar to basePath)